docs/core/tutorials/with-visual-studio.md

```diff
@@ -1,82 +1,93 @@
 ---
-title: Build a C# Hello World application with .NET Core in Visual Studio 2017
-description: Learn how to build a simple .NET Core console application with C# using Visual Studio 2017.
+title: Create a C# Hello World application with .NET Core in Visual Studio 2019
+description: Learn how to create your first .NET Core console application with C# using Visual Studio 2019.
 author: BillWagner
 ms.author: wiwagn
-ms.date: 09/13/2017
+ms.date: 12/03/2019
 ms.custom: "vs-dotnet, seodec18"
```

Comment:
```suggestion
ms.custom: "vs-dotnet, seodec18 updateeachrelease"
```

Reply:
vs-dotnet is the one being used for vs releases. But I'm happy to change that one or reconsider this suggestion if we prefer the updateeachrelease instead (or a different one)